### PR TITLE
Gamedrake

### PR DESCRIPTION
**Pull request by slimmoe**

### CODE REVIEW

File: gamedrake/metadata.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "author": {
+      "name": "slimmoe",
+      "id": "686244278304899077"
+    },
+    "service": "Gamedrake",
+    "description": {
+        "en": "Game Drake is the platform for players to learn about games.",
+        "tr": "Gamedrake, oyunlar hakkında bilgi edinebileceğiniz oyuncu platformu." 
+       },
+    "url": ["gamedrake.com"],
```
Comment (eggsywashere) [inline gamedrake/metadata.json:11]: Please change it to string instead of an array (remove `[` and `]`)

File: gamedrake/metadata.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "author": {
+      "name": "slimmoe",
```
Comment (eggsywashere) [inline gamedrake/metadata.json:3]: Enter your full name here. You can use spaces (`Slim Moe`)

File: gamedrake/presence.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+var presence = new Presence({
+    clientId: "707525132322865152"
+  });
+  
+  presence.on("UpdateData", () => {
+    let presenceData = {
```
Comment (eggsywashere) [inline gamedrake/presence.js:6]: No `largeImageKey`? Check documentation and use an image.

File: gamedrake/presence.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    };
+  
+    if (window.location.href.includes("https://gamedrake.com/editor-ol/")) {
+      presenceData.details = "Editör Ol sayfasını görüntülüyor:";
+    } else if (window.location.href.includes("ro-the-bot.ga")) {
+      if (window.location.pathname.toLowerCase() === "/") {
+        presenceData.details = "Anasayfayı görüntülüyor:";
+        presenceData.state = "Anasayfa";
+      }
+      if (window.location.pathname.toLowerCase().includes("/k/oyun-haberleri")) {
```
Comment (eggsywashere) [inline gamedrake/presence.js:16]: İt's better if you use `else if`s here and the ones after this.

**Comment by Cairo2k18** [comment]
Please close this in favour of #1507

**Pull request closed without merging**